src/pile.js

```diff
@@ -19,8 +23,8 @@ export const MODE_SELECTED = 'Selected';
  * @param {object}   options.store - Redux store
  */
 const createPile = ({ initialItem, render, id, pubSub, store }) => {
-  const itemsById = new Map();
```

Comment:
If we use `Map()` to store the item instances, when we pass `(item, pile, i)` to user's callback function, the `i` would be the key in the map, i.e. the id of the item, instead of the position of the item in the pile. 
Also, it seems like storing items by their id is not necessary for now, we just need a list of item instance, so I just use an array to store the items.

Reply:
> Also, it seems like storing items by their id is not necessary for now, we just need a list of item instance, so I just use an array to store the items.

Thanks for the explanation.

Actually, this is going to become important soon. One of the things we need to test in the near future is whether our library can properly deal with updating items. I.e., the rendering or associated data of items might change and then we need to know which item was actually updated.

In the future we might need both, a `Map()` and an `Array` but for now this is fine.